docs/infrastructure.md

```diff
@@ -32,6 +32,8 @@ We have tested the following options:
 
     We recommend using GraphDB if these restrictions are not a blocker.
 
+    **Note: You do not need to download GraphDB from the official website for the setup steps below.**
+
```

Comment:
How about we put this into a warning or info box instead. We already have many of them on the page, so to make some room, how about we also delete the one about RDF being a niche thing:

![image](https://github.com/neurobagel/documentation/assets/1302022/7e63cc19-882b-49d4-b8e3-c5d1432aee04)

i.e.
- make current "RDF is niche" box go away
- put new warning in a box